packages/spor-react/src/input/FormErrorMessage.tsx

```diff
@@ -32,7 +32,10 @@ export type FormErrorMessageProps = {
  *
  * @see https://spor.vy.no/komponenter/skjemaelementer
  */
-export const FormErrorMessage = ({ children }: FormErrorMessageProps) => {
+export const FormErrorMessage = ({
```

Comment:
`FormErrorComponent` hadde ikke støtte for å sende inn `BoxProps`, og det trenger man om man vil bruke en feilmelding på PhoneNumberInput (flytte den litt til høyre, avhengig av hva som er feil).